ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.html

```diff
@@ -1,6 +1,6 @@
-<mat-accordion hideToggle="true" multi="true" displayMode="flat">
+<mat-accordion hideToggle="true" multi="true" displayMode="flat" >
     <ng-container *ngFor="let item of questions | translate | slice : 0 : questionsCount;">
-        <mat-expansion-panel #panel>
+        <mat-expansion-panel #panel [class.darkTheme]="darkTheme">
```

Comment:
I'm not sure darkTheme is a good choise for the name here since we don't make background dark/foreground light, we just have foreground slightly darker because of parent has dark background, so might be confusing, consider darkerColor os smth

Reply:
Renamed.